src/cpp/configuration/DiscoveryServerParticipantConfiguration.cpp

```diff
@@ -77,6 +77,45 @@ std::vector<DiscoveryServerConnectionAddress> DiscoveryServerParticipantConfigur
     return result;
 }
 
+std::map<std::string, std::string> DiscoveryServerParticipantConfiguration::tls_configuration() const
+{
+    std::map<std::string, std::string> result;
+
+    if (raw_configuration_[TLS_TAG])
+    {
+        if (!raw_configuration_[TLS_TAG].IsMap() && !raw_configuration_[TLS_TAG].IsNull())
```

Comment:
Probably TLS should not be empty ¿?

Reply:
Don't know, I thought it's not enough for breaking.